tests/v1/worker/test_gpu_model_runner.py

```
@@ -65,6 +65,93 @@
+        def __init__(self, cache_ref):
+            self.cache_ref = cache_ref
+            self.instances.append(self)
+
+        @classmethod
+        def reset_kv_cache_binding_state(cls):
+            cls.reset_calls += 1
+            assert all(
+                actual_cache is profiling_cache
+                for actual_cache, profiling_cache in zip(
+                    runner.kv_caches, profiling_caches
+                )
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)